src/mongo/util/net/sock.h

```diff
@@ -73,7 +73,9 @@ extern const int portRecvFlags;
 inline void closesocket(int s) {
     close(s);
 }
-const int INVALID_SOCKET = -1;
+#ifndef INVALID_SCOKET
+#define INVALID_SOCKET = -1;
+#endif
 typedef int SOCKET;
```

Comment:
```suggestion
#endif

typedef int SOCKET;
```

nit: good to add a linebreak, for readability here